docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -291,13 +291,20 @@ WARNING: This only works when your application is running inside a Google Cloud
 
 === Using Cloud SQL with native executables
 
-When generating native executables, you must also mark `jnr.ffi.provider.jffi.NativeFinalizer$SingletonHolder` as runtime initialized.
+When generating native executables, you must mark `jnr.ffi.provider.jffi.NativeFinalizer$SingletonHolder` as runtime initialized.
 
 [source,properties]
 ----
 quarkus.native.additional-build-args=--initialize-at-run-time=jnr.ffi.provider.jffi.NativeFinalizer$SingletonHolder
 ----
 
+Additionally, starting with `com.google.cloud.sql:postgres-socket-factory:1.17.0`, you must also mark `com.kenai.jffi.internal.Cleaner` as runtime initialized.
+
+[source,properties]
+----
+quarkus.native.additional-build-args=--initialize-at-run-time=jnr.ffi.provider.jffi.NativeFinalizer$SingletonHolder\,com.kenai.jffi.internal.Cleaner
```

Comment:
There is a small typo here, there should be double backslashes to escape the comma. See: https://github.com/quarkusio/quarkus/issues/6710#issuecomment-577066807

Reply:
Thank you @daniel-pettersson-pricer. Fixed.

Reply:
@gsmet it's not the first time we asked ourself about an extension for CloudSQL but the issue is that as it would not be able to test on a real instance, this would probably not catch this kind of errors so I'm not sure it's worth the work.

Reply:
Well, that would be a good way to capitalize on things. I agree it's not going to magically fix the reported issues but at least people could get the latest fixes just by updating.

Now it's just an idea.

Reply:
Tracked here: https://github.com/quarkiverse/quarkus-google-cloud-services/issues/696